vs2010.vcxproj.static.template

```diff
@@ -189,6 +189,7 @@
     <ClCompile Include=".\src\bb.c" />
     <ClCompile Include=".\src\error.c" />
     <ClCompile Include=".\src\jnilib.c" />
+    <ClCompile Include=".\src\nativestaticallyreferencedjnimethods.c" />
```

Comment:
this is consistent with transport-native-epoll ... and its an internal class so no-one will know (shhh) :)